### PR TITLE
Fix info for svg `<style>` element

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

`media` attribute ( for `<style>` element) is not animatable
update type for `<style>` attributes

### Additional details

https://svgwg.org/svg2-draft/styling.html#StyleElement

### CODE REVIEW

File: files/en-us/web/svg/element/style/index.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ svg {
 
 - {{SVGAttr("type")}}
   - : This attribute defines type of the style sheet language to use as a media type string.
-    _Value type_: [**\<string>**](/en-US/docs/Web/CSS/string); _Default value_: `text/css`; _Animatable_: **no**
+    _Value type_: [**`<content-type>`**](https://www.ietf.org/rfc/rfc2046.txt); _Default value_: `text/css`; _Animatable_: **no**
```
Comment (hamishwillee) [inline files/en-us/web/svg/element/style/index.md:45]: I'm not sure about this change as we prefer not to link to specs. Could we instead link to https://developer.mozilla.org/en-US/docs/Glossary/MIME_type and call it a`<media-type>` (the spec does)?

The rest looks good.

Comment (hamishwillee) [reply files/en-us/web/svg/element/style/index.md:45]: P.S, mostly I think the linked doc is much more accessible than the spec.

Comment (hamishwillee) [reply files/en-us/web/svg/element/style/index.md:45]: P.S. Just FYI @skyclouds2001 I am here Monday, maybe Tuesday but then will be away a lot until Jan 20. So if something gets auto-assigned to me, you will either have to find someone else to look at it or wait. I don't want to hold you up!

Comment (skyclouds2001) [inline files/en-us/web/svg/element/style/index.md:45]:
```suggestion
    _Value type_: [**`<media-type>`**](/en-US/docs/Glossary/MIME_type); _Default value_: `text/css`; _Animatable_: **no**
```

**Review by hamishwillee — APPROVED** [review]
Thanks very much @skyclouds2001 !

**Pull request merged**